README.md

```diff
@@ -148,11 +143,14 @@ one of the examples in this SDK, try:
 
 1. ``cd examples/stocks``
 
-2. ``pub get`` to set up a copy of the sky package in the app directory.
+2. ``pub get`` to set up a copy of the sky package in the app directory. For Windows users, you will need to copy packages\sky\apks\SkyDemo.apk to examples\stocks\packages\apks\
 
 3. ``./packages/sky/sky_tool start --install``
    The --install flag is only necessary to install SkyDemo.apk if not already
    installed on the device.
+   
+   A this stage, there is a little difference if you are a Windows user. You must use the following command line :
```

Comment:
Windows users should be able to do the / to \ translation in their heads. :)  We can add a sky_tool.bat file if that's helpful.